packages/compass-crud/electron/renderer/components/query-bar.jsx

```diff
@@ -0,0 +1,63 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+
+class QueryBar extends React.Component {
```

Comment:
This file was removed on main, we can remove it here as well, maybe it was checked in by accident?

Reply:
Probably some merge conflict that wasn't properly handled. Will remove it!